File: CS/NetFramework/net48/01_Controls/Controls.cs
using System;
using System.Windows.Forms;

namespace InputManWin13_Demo._01_Controls
{
    public partial class Controls : InputManWin13_Demo.DemoBase
    {
        public Controls()
        {
            InitializeComponent();
            InitializeForm();

        }
        private void InitializeForm()
        {
            // ***** 初期設定

            // xmlファイルからデータ取得（gcMaskedComboBox）
            System.String filePath = Application.StartupPath;
            this.dataSet1.ReadXml(filePath + "\\Resources\\pref.xml");
            gcMaskedComboBox1.DataSource = dataSet1.Tables[0];
            gcMaskedComboBox1.SelectedIndex = -1;

            for (int i = 1; i < gcMaskedComboBox1.ListColumns.Count; i++)
            {
                gcMaskedComboBox1.ListColumns[i].AutoWidth = true;
            }
        }

    }
}

File: VB/NetFramework/net48/01_Controls/Controls.vb
Imports System.Windows.Forms

Namespace _01_Controls
    Partial Public Class Controls
        Inherits InputManWin13_Demo.DemoBase
        Public Sub New()
            InitializeComponent()
            InitializeForm()
        End Sub

        Private Sub InitializeForm()
            ' xmlファイルからデータ取得
            Dim filePath As System.String = Application.StartupPath
            Me.DataSet1.ReadXml(filePath & "\Resources\pref.xml")
            GcMaskedComboBox1.DataSource = DataSet1.Tables(0)
            GcMaskedComboBox1.SelectedIndex = -1

            For i As Integer = 1 To GcMaskedComboBox1.ListColumns.Count - 1
                GcMaskedComboBox1.ListColumns(i).AutoWidth = True
            Next
        End Sub


    End Class
End Namespace
